tests/main/svcs-disable-install-hook/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
+
+execute: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-svcs-disable-install-hook
+
+    for service in simple forking; do
+        echo "Verify that the $service service isn't running"
+        snap services | MATCH "test-snapd-svcs-disable-install-hook\\.$service\\s\+disabled\\s\+inactive"
```

Comment:
This works `MATCH "test-snapd-svcs-disable-install-hook\\.$service\\s+disabled\\s+inactive"`

Reply:
Indeed it does! This test is now passing. I think I was confused because the escaping here is really a bash string inside a yaml string...